### PR TITLE
[ja] Rename file and update concepts/architecture/control-plane-node-communication.md

### LINKED ISSUE #35470: [ja] Translate concepts/architecture/control-plane-node-communication into Japanese
**This is a Feature Request**

**What would you like to be added**

Translate [concepts/architecture/control-plane-node-communication/](https://kubernetes.io/docs/concepts/architecture/control-plane-node-communication/) into Japanese.

**Why is this needed**

This page has not been translated into Japanese.

**Comments**

/language ja

### PR DESCRIPTION
**Pull request by t-inu**
fixes #35470

ref: https://github.com/kubernetes/website/pull/20265
Rename the file from `master-node-communication.md` to `control-plane-node-communication.md`.
And update the contents.

Preview:
https://deploy-preview-36632--kubernetes-io-main-staging.netlify.app/ja/docs/concepts/architecture/control-plane-node-communication/

en:
https://kubernetes.io/docs/concepts/architecture/control-plane-node-communication/
https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/architecture/control-plane-node-communication.md

### CODE REVIEW

File: content/ja/docs/concepts/architecture/control-plane-node-communication.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+本ドキュメントは、APIサーバーとKubernetesクラスター間の通信経路をまとめたものです。
+その目的は、信頼できないネットワーク上(またはクラウドプロバイダー上の完全なパブリックIP)でクラスターが実行できるよう、ユーザーがインストールをカスタマイズしてネットワーク構成を強固にできるようにすることです。
+
+<!-- body -->
+
+## ノードからコントロールプレーンへの通信 {#node-to-control-plane}
+
+Kubernetesには「ハブアンドスポーク」というAPIパターンがあります。ノード(またはノードが実行するPod)からのすべてのAPIの使用は、APIサーバーで終了します。他のコントロールプレーンコンポーネントは、どれもリモートサービスを公開するようには設計されていません。APIサーバーは、1つ以上の形式のクライアント[認証](/ja/docs/reference/access-authn-authz/authentication/)が有効になっている状態で、セキュアなHTTPSポート(通常は443)でリモート接続をリッスンするように設定されています。
+特に[匿名リクエスト](/ja/docs/reference/access-authn-authz/authentication/#anonymous-requests)や[サービスアカウントトークン](/ja/docs/reference/access-authn-authz/authentication/#service-account-token)が許可されている場合は、1つ以上の[認証](/ja/docs/reference/access-authn-authz/authentication/)形式を有効にする必要があります。
```
Comment (atoato88) [inline content/ja/docs/concepts/architecture/control-plane-node-communication.md:19]:
```suggestion
特に[匿名リクエスト](/ja/docs/reference/access-authn-authz/authentication/#anonymous-requests)や[サービスアカウントトークン](/ja/docs/reference/access-authn-authz/authentication/#service-account-token)が許可されている場合は、1つ以上の[認可](/docs/reference/access-authn-authz/authorization/)形式を有効にする必要があります。
```
"authorization" is "認可" in Japanese and [that document](https://kubernetes.io/docs/reference/access-authn-authz/authorization/) on kubernetes.io isn't translated to Japanese.
We should use a linkt to EN contents. :thinking:

File: content/ja/docs/concepts/architecture/control-plane-node-communication.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+ノードは、有効なクライアント認証情報とともに、APIサーバーに安全に接続できるように、クラスターのパブリックルート証明書をプロビジョニングする必要があります。適切なやり方は、kubeletに提供されるクライアント認証情報が、クライアント証明書の形式であることです。kubeletクライアント証明書の自動プロビジョニングについては、[kubelet TLSブートストラップ](/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/)を参照してください。
```
Comment (atoato88) [inline content/ja/docs/concepts/architecture/control-plane-node-communication.md:21]:
```suggestion
ノードは、有効なクライアント認証情報とともに、APIサーバーに安全に接続できるように、クラスターのパブリックルート証明書でプロビジョニングされる必要があります。適切なやり方は、kubeletに提供されるクライアント認証情報が、クライアント証明書の形式であることです。kubeletクライアント証明書の自動プロビジョニングについては、[kubelet TLSブートストラップ](/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/)を参照してください。
```

File: content/ja/docs/concepts/architecture/control-plane-node-communication.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+コントロールプレーン(APIサーバー)からノードへの主要な通信経路は2つあります。
+1つ目は、APIサーバーからクラスター内の各ノードで実行されるkubeletプロセスへの通信経路です。
+2つ目は、APIサーバーの _プロキシー_ 機能を介した、APIサーバーから任意のノード、Pod、またはサービスへの通信経路です。
+
+### APIサーバーからkubeletへの通信 {#api-server-to-kubelet}
+
+APIサーバーからkubeletへの接続は、以下の目的で使用されます:
+
+* Podのログの取得。
+* 実行中のPodへのアタッチ(通常はkubectlを使用)。
```
Comment (atoato88) [inline content/ja/docs/concepts/architecture/control-plane-node-communication.md:41]:
```suggestion
* 実行中のPodへのアタッチ(通常は`kubectl`を使用)。
```

File: content/ja/docs/concepts/architecture/control-plane-node-communication.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+最後に、kubelet APIを保護するために、[Kubelet認証/認可](/docs/reference/access-authn-authz/kubelet-authn-authz/)を有効にする必要があります。
+
+### APIサーバーからノード、Pod、サービスへの通信 {#api-server-to-nodes-pods-and-services}
+
+APIサーバーからノード、Pod、またはサービスへの接続は、デフォルトで平文のHTTP接続になるため、認証も暗号化もされません。API URL内のノード、Pod、サービス名に`https:`を付けることで、セキュアなHTTPS接続を介して実行できますが、HTTPSエンドポイントから提供された証明書を検証したり、クライアント認証情報を提供したりすることはありません。そのため、接続の暗号化はされますが、完全性の保証はありません。これらの接続を、信頼されていないネットワークやパブリックネットワークを介して実行するのは、**現在のところ安全ではありません**。
+
+### SSHトンネル {#ssh-tunnels}
+
+Kubernetesは、コントロールプレーンからノードへの通信経路を保護するために、SSHトンネルをサポートしています。この構成では、APIサーバーがクラスター内の各ノードへのSSHトンネルを開始(ポート22でリッスンしているSSHサーバーに接続)し、kubelet、ノード、Pod、またはサービス宛てのすべてのトラフィックをトンネル経由で渡します。
+このトンネルにより、ノードが実行されているネットワークの外部にトラフィックが公開されないようになります。
```
Comment (atoato88) [inline content/ja/docs/concepts/architecture/control-plane-node-communication.md:60]:
```suggestion
このトンネルにより、ノードが稼働するネットワークの外部にトラフィックが公開されないようになります。
```

**Review by atoato88 — COMMENTED** [review]
Please check comments.

File: content/ja/docs/concepts/architecture/control-plane-node-communication.md (hunk 1 above)
Comment (t-inu) [reply content/ja/docs/concepts/architecture/control-plane-node-communication.md]: Thank you for finding my mistake!

**Comment by t-inu** [comment]
@atoato88 Fixed what you pointed out.

**Comment by atoato88** [comment]
/lgtm
Thanks you for quick update.

**Pull request merged**